python/chemiscope/input.py

```diff
@@ -145,9 +145,11 @@ def write_input(path, frames, meta=None, properties=None, cutoff=None):
 
     .. code-block:: python
 
+        import ase
         from ase import io
         import numpy as np
         import sklearn
+        from sklearn import decomposition
```

Comment:
Why is this necessary? With the import of sklearn, we shouldn't need the explicit decomposition import.

Reply:
At least locally, I get this error:

```
>>> import sklearn
>>> sklearn.decomposition.PCA
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'sklearn' has no attribute 'decomposition'
>>>
```

This was also reported by Alex in #108